src/ontology/cl-edit.owl

```diff
@@ -7032,9 +7031,10 @@ SubClassOf(obo:CL_0000419 obo:CL_0000036)
 
 # Class: obo:CL_0000420 (syncytial epithelial cell)
 
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0000420 "An epithelial cell that forms a syncytium, which is a multinucleated cell resulting from the fusion of multiple cells.")
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000420 "Wikipedia:Syncytium")
 AnnotationAssertion(rdfs:label obo:CL_0000420 "syncytial epithelial cell")
-SubClassOf(obo:CL_0000420 obo:CL_0000066)
```

Comment:
I think this shouldn't have been removed.

Reply:
I have added them